### DISCUSSION TITLE
ドキュメントの「API Reference」に記述のエンドポイントは、全てaccess_tokenでアクセスできるのでしょうか？

### DISCUSSION THREAD

**Discussion by funakura** [growilabs/growi#9093] [Help / Questions] [answered]
[API Reference](https://docs.growi.org/en/api/rest-v3.html)に記載のエンドポイントは、全てAPI-Key（access_token）でアクセスできるのでしょうか？
例えば、グループ情報の取得の`GET: https://demo.growi.org/_api/v3/user-groups`にアクセスしたら「Forbidden」がか行ってきます。

**Comment by yuki-takei** [comment, marked as answer]
申し訳ないのですが API Reference はきちんとメンテナンスされていないため、正しくない情報であることも多いです。
`/_api/v3/user-groups` に関しては access token には対応していません。

**Comment by funakura** [comment]
回答ありがとうございます。
了解しました。
Express,Next.jsは不勉強ですが、 ソースコードを見ることにします。
APIでグループの管理を行いたいです。できたらaccess token対応してもらえると嬉しいです。
